### PR TITLE
15_3_3 Main program loop

### PR DESCRIPTION
**Pull request by sergeroshe**

### CODE REVIEW

File: 15_mini_project_work/3_magic_ball_game.py [hunk 1]
```diff
@@ -8,11 +8,30 @@
...
+
+def game_run():
+    game_is_on = True
+    while game_is_on:
+        question = input(ENTER_QUESTION_PROMPT).lower()
+        answer = choice(ANSWERS)
+        print(answer)
+
+        continue_game = input(CONTINUE_GAME_PROMPT)
+        if continue_game != YES_RESPONSE:
```
Comment (YanumVlad) [inline 15_mini_project_work/3_magic_ball_game.py:26]: Try to guess yourself - what is wrong here

**Review by YanumVlad — APPROVED** [review]

**Review by YanumVlad — CHANGES_REQUESTED** [review]
Let's make is user-friendly:
COntinue game on pressing ANY button.
Exit - if there is explicit N button press
So, user have the variety of options to continue (not just "1")

File: 15_mini_project_work/3_magic_ball_game.py [hunk 2]
```diff
@@ -8,11 +8,30 @@
 USER_NAME_PROMPT = 'Введи свое имя:\n'
 EXCLAMATION_SIGN = '!'
 USER_NAME_GREETING = 'Добро пожаловать, '
+CONTINUE_GAME_PROMPT = 'Хочешь задать еще один вопрос?\nНажми "1", затем ENTER,' \
+                       ' если ДА\nНажми любую клавишу, затем ENTER, если НЕТ\n'
+ENTER_QUESTION_PROMPT = 'Введи свой вопрос:\n'
+YES_RESPONSE = '1'
```
Comment (YanumVlad) [inline 15_mini_project_work/3_magic_ball_game.py:14]: Let's make is user-friendly:
COntinue game on pressing ANY button.
Exit - if there is explicit N button press
So, user have the variety of options to continue (not just "1")

**Review by YanumVlad — CHANGES_REQUESTED** [review]
Let's make is user-friendly:
COntinue game on pressing ANY button.
Exit - if there is explicit N button press
So, user have the variety of options to continue (not just "1")

File: 15_mini_project_work/3_magic_ball_game.py (hunk 1 above)
Comment (sergeroshe) [reply 15_mini_project_work/3_magic_ball_game.py:26]: I tried. For this step's requirement I don't see any problem so far.

Comment (sergeroshe) [reply 15_mini_project_work/3_magic_ball_game.py:26]: I tried. For this step's requirement I don't see any problem so far.

File: 15_mini_project_work/3_magic_ball_game.py (hunk 2 above)
Comment (sergeroshe) [reply 15_mini_project_work/3_magic_ball_game.py:14]: Got it. I will do it.

**Review by YanumVlad — APPROVED** [review]

**Pull request merged**